src/IIIF/IIIF/Presentation/HtmlSanitiser.cs

```diff
@@ -83,7 +115,25 @@ public static string SanitiseHtml(this string propertyValue, bool ignoreNonHtml
         }
         workingString = $"<{nonHtmlWrappingTag}>{workingString}</{nonHtmlWrappingTag}>";
 
-        return Sanitizer.Sanitize(workingString);
+        return Sanitise(workingString);
+    }
+
+    /// <summary>
+    /// Sanitise provided text, removing any invalid tags/attributes etc
+    /// </summary>
+    private static string Sanitise(string toSanitise)
+    {
+        var workingString = Sanitizer.Sanitize(toSanitise);
+        
+        // At this point "<span><ul><li>One</li><li>Two</li><li>Three Four</li><ul></span>" would
+        // be "<span>~||~One~||~~||~Two~||~~||~Three Four~||~</span>", clean that up to
+        // "<span>One Two Three Four</span>"
+        return workingString
+            .Replace($"{TagReplacement}<", "<")
+            .Replace($">{TagReplacement}", ">")
+            .Replace($"{TagReplacement}{TagReplacement}", TagReplacement)
+            .Replace(TagReplacement, " ")

```

Comment:
I think this should be conditional on if the tag is a block-level or inline-level element. Inline elements, such as `<strong>some</strong>thing` wouldn't produce a space if rendered in HTML.